### DISCUSSION TITLE
A 'delivered' option for ideas

### DISCUSSION THREAD

**Discussion by joebudi** [community/community#4076] [Discussions]
Our community raises ideas, we take these ideas on board and implement them. There's no way to mark that discussion as done. With q&a you have the 'answered' action, which works perfectly, but for ideas, it would be nice to have a done/delivered action that highlights to the readers, this idea was delivered and is now live - this shows the community you listen to feedback.

**Comment by laymonage** [comment]
What about using labels? It's also used in this repository, for example: https://github.com/github/feedback/discussions?discussions_q=label%3A%22%F0%9F%9A%80+shipped%22+

**Reply by joebudi** [reply]
I suppose this could work to a degree, but on the mobile app, the user does not see the label unless they are in the discussion.

Also, am I right in thinking, labels are shared amongst issues, PRs, and Discussions?

**Reply by laymonage** [reply]
Ah, I see. Maybe you can submit a feedback for the mobile app or wait for some hubbers to weigh in on this suggestion. Might also be related to https://github.com/github/feedback/discussions/3097.

> Also, am I right in thinking, labels are shared amongst issues, PRs, and Discussions?

Yes, they are shared.
